.github/workflows/deploy-docs.yml

```diff
@@ -0,0 +1,21 @@
+name: Build and Deploy Docs
+
+on:
+  release:
+    types: [published]
+
+# Sets permissions of the GITHUB_TOKEN to allow deployment to GitHub Pages
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+# Allow only one concurrent deployment, skipping runs queued between the run in-progress and latest queued.
+# However, do NOT cancel in-progress runs as we want to allow these production deployments to complete.
+concurrency:
+  group: "pages"
+  cancel-in-progress: false
```

Comment:
I found that this stanza actually causes the workflow to be cancelled, since it also exists in the common actions version. I think best is to leave it in the common actions workflow, so each caller doesn't have to worry about it.

Reply:
Sounds reasonable. 👍